tests/run-make/extern-flag-disambiguates/rmake.rs

```diff
@@ -0,0 +1,30 @@
+//@ ignore-cross-compile
+
+use run_make_support::{cwd, run, rustc};
+
+// Attempt to build this dependency tree:
+//
+//    A.1   A.2
+//     |\    |
+//     | \   |
+//     B  \  C
+//      \ | /
+//       \|/
+//        D
+//
+// Note that A.1 and A.2 are crates with the same name.
+
+// original Makefile at https://github.com/rust-lang/rust/commit/cc3c8bbfaf5af19caf3deb131a995a65ca4674f9#diff-23d3f6bcee0c974ae7c793ab6f87c454cffd1f5c9ec17ce86aaf2cf8bc6e5397
```

Comment:
I think this might fail tidy's maximum 100 character line length, my apologies. One of the following could be a suitable replacement:

https://github.com/rust-lang/rust/issues/14469
https://github.com/rust-lang/rust/pull/15319

Reply:
done